content/awell-orchestration/api-reference/queries/get-published-pathways.mdx

```diff
@@ -32,11 +32,27 @@ query GetPublishedPathwayDefinitions {
           value
         }
       }
+      track_definitions {
+        id
+        title
+        can_trigger_manually
+      }
+      dataPointDefinitions {
+        # @deprecated use data_point_definitions
+        id
+        # ...
+      }
```

Comment:
This is confusing and I don't understand what we are saying here.

Reply:
Make it simple for the user: only document what we expect the user to use moving forward. Given it's not a breaking change, there's no risk of removing the old documentation completely.

Or just something like:
```graphql
{
data_point_definitions { # previously dataPointDefinitions
}